feature-libs/user/profile/components/close-account/components/close-account-modal/close-account-modal.component.ts

```diff
@@ -80,7 +80,7 @@ export class CloseAccountModalComponent implements OnInit {
           GlobalMessageType.MSG_TYPE_CONFIRMATION
         );
       });
-    this.routingService.go({ cxRoute: 'home' });
+    this.routingService.go({ cxRoute: 'logout' });
```

Comment:
This logs the user out but the user appears to stay on the close account page after the account is closed. Shouldn't we redirect back to the home or login after the account is closed? The e2e test should verify this behaviour.
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/11735817/198515951-7aa74d66-9aca-4be6-b1f6-df79d558ac18.png">

Reply:
@Zeyber I'm seeing the correct behavior the user is logged out and redirected to the homepage 
![image](https://user-images.githubusercontent.com/25436835/204437829-74efb786-d5b7-42cc-be4e-27f20a82cae5.png)

Reply:
This may be the case with manual testing but it is not apparent with the current e2e test. This test is known to be flaky as-is which is why we need to be meticulous in verifying correct behaviour.

FYI: Timing is a common issue with causing test flakes so if a test completes too quickly without verifying all steps completed, requests may be cancelled before the test finishes and cause issues for the next test.

Reply:
You will also need to handle the breaking change here with the `featureConfigService.isLevel()` method.